micrometer-spring-legacy/build.gradle

```diff
@@ -44,6 +44,7 @@ dependencies {
 
     // to validate auto-completion on configuration properties
     annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
+    annotationProcessor 'org.springframework.boot:spring-boot-autoconfigure-processor'
```

Comment:
This one is present in the `1.0.x` branch. I am unsure if it has been lost in a merge to master (`1.1.x`) or if `1.0.x` hasn't been merged since it's introduction. (Looks like it got lost If I get it right.)

Reply:
On afterthough, should this addition to build.gradle be aligned with the `1.0.x` line so that the files don't drift that much?